firmware/pll.c

```diff
@@ -70,6 +70,8 @@ void pll_config_for_clock(int freq)
 	 * Reproducible both with DRP and initial reconfiguration.
 	 * Until this spartan6 weirdness is sorted out, just stick to 20x.
 	 */
+
+	(void) pll_config_10x[0]; // Dummy access to suppress to suppress -Werror=unused-const-variable.
```

Comment:
I think we want to use something like https://stackoverflow.com/questions/4851075/universally-compiler-independent-way-of-implementing-an-unused-macro-in-c-c

Reply:
Or https://stackoverflow.com/questions/7090998/portable-unused-parameter-macro-used-on-function-signature-for-c-and-c

Reply:
Clearly, there is no universal solution :P. the `sizeof` solution [causes warnings](https://stackoverflow.com/questions/4851075/universally-compiler-independent-way-of-implementing-an-unused-macro-in-c-c#comment22169080_4851173) on newer compilers.

I think unused vars are infrequent enough that I prefer my solution with works just fine for this single use case. That said, if you _really_ want a macro, `__attribute__(unused)` should be fine, since `gcc`/`clang` is currently the only compiler for all CPUs we support. But I would prefer to gate it behind a macro like the second answer (just in case).

What header in HDMI2USB/litex-buildenv should such a macro go?